nss-day-cohort-45/tabloidmvc-missing-persons
# View User Profile

# View User Profile

### How To Test
1. Fetch branch
2. Run application and log in as an admin
2. Click User Profiles in the navigation bar
3. Check to see if you can view all the users. Make sure they are in alphabetical order, from top to down.
